test-utils/state-viewer/src/main.rs

```diff
@@ -33,51 +34,59 @@ fn to_printable(blob: &[u8]) -> String {
     }
 }
 
-fn kv_to_state_record(key: Vec<u8>, value: DBValue) -> StateRecord {
+fn kv_to_state_record(key: Vec<u8>, value: DBValue) -> Option<StateRecord> {
     let column = &key[0..1];
     match column {
         col::ACCOUNT => {
             let separator = (1..key.len()).find(|&x| key[x] == ACCOUNT_DATA_SEPARATOR[0]);
             if separator.is_some() {
-                StateRecord::Data { key: to_base64(&key), value: to_base64(&value) }
+                Some(StateRecord::Data { key: to_base64(&key), value: to_base64(&value) })
             } else {
                 let account = Account::try_from_slice(&value).unwrap();
-                StateRecord::Account {
+                Some(StateRecord::Account {
                     account_id: String::from_utf8(key[1..].to_vec()).unwrap(),
                     account: account.into(),
-                }
+                })
             }
         }
-        /*
-        col::CALLBACK => {
-            let callback: Callback = Decode::decode(&value).unwrap();
-            StateRecord::Callback { id: key[1..].to_vec(), callback }
-        }
-        */
-        col::CODE => {
-            StateRecord::Contract { account_id: to_printable(&key[1..]), code: to_base64(&value) }
-        }
+        col::CODE => Some(StateRecord::Contract {
+            account_id: String::from_utf8(key[1..].to_vec()).unwrap(),
+            code: to_base64(&value),
+        }),
         col::ACCESS_KEY => {
             let separator = (1..key.len()).find(|&x| key[x] == col::ACCESS_KEY[0]).unwrap();
             let access_key = AccessKey::try_from_slice(&value).unwrap();
             let account_id = String::from_utf8(key[1..separator].to_vec()).unwrap();
             let public_key = PublicKey::try_from(&key[(separator + 1)..]).unwrap();
-            StateRecord::AccessKey {
+            Some(StateRecord::AccessKey {
                 account_id,
                 public_key: public_key.into(),
                 access_key: access_key.into(),
-            }
+            })
+        }
+        col::RECEIVED_DATA => {
+            let data = ReceivedData::try_from_slice(&value).unwrap().data;
+            let separator = (1..key.len()).find(|&x| key[x] == ACCOUNT_DATA_SEPARATOR[0]).unwrap();
+            let account_id = String::from_utf8(key[1..separator].to_vec()).unwrap();
+            let data_id = CryptoHash::try_from(&key[(separator + 1)..]).unwrap();
+            Some(StateRecord::ReceivedData { account_id, data_id: data_id.into(), data })
+        }
+        col::POSTPONED_RECEIPT_ID => None,
+        col::PENDING_DATA_COUNT => None,
+        col::POSTPONED_RECEIPT => {
+            let receipt = Receipt::try_from_slice(&value).unwrap();
+            Some(StateRecord::PostponedReceipt(receipt.into()))
         }
         _ => unreachable!(),
```

Comment:
what's left? let's either remove or add explicit items here.

Reply:
7..255 columns. We currently, don't use enum for this.

Reply:
I'll change it to enums in other PR